docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,819 @@
+  --colors-primary-100: 224 247 235;
+  --colors-primary-200: 187 235 210;
+  --colors-primary-300: 135 216 177;
+  --colors-primary-400: 77 192 140;
+  --colors-primary-500: 45 165 116;
+  --colors-primary-600: 34 134 95;
+  --colors-primary-700: 31 110 80;
+  --colors-primary-800: 30 86 65;
+  --colors-primary-900: 23 64 49;
+}
+```
+::
+
+### 3. Using Store-Specific Styles
+
+Now you can use these variables in your components through Tailwind's configuration:
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+First, update your Tailwind config:
+
+```ts
+// packages-end-user/tailwind-config/src/nextjs.ts
+import { tailwindConfig } from "@storefront-ui/react/tailwind-config";
+import type { Config } from "tailwindcss";
+import plugin from "tailwindcss/plugin";
+
+const config: Config = {
+  // ...
+  plugins: [
+    // ...
+    plugin(({ addVariant }) => {
+      // Add variant for electronics store
+      addVariant("electronics", ".electronics &");
+      // Add variant for apparel store
+      addVariant("apparel", ".apparel &");
+    }),
+  ],
+  // ...
+};
+export default config;
+```
+
+Then use these variants (`electronics:`, `apparel:`, etc.) in your components:
+
+```tsx
+// apps/storefront-unified-nextjs/components/ui/navbar-top.tsx
+export default function NavbarTop({ children, className }: NavbarTopProps) {
+  return (
+    <header
+      className={classNames(
+        'sticky top-0 z-40 flex h-14 bg-primary-800',
+        'electronics:!bg-primary-600',
+        'apparel:!bg-primary-400',
+        'md:-top-5 md:h-20 md:pt-2.5',
+        className
+      )}
+    >
+      {children}
+    </header>
+  );
+}
+```
+
+#tab-2
+First, update your Tailwind config:
+
+```ts
+// packages-end-user/tailwind-config/src/nuxt.ts
+import type { Config } from "tailwindcss";
+import plugin from "tailwindcss/plugin";
+
+export default {
+  // ...
+  plugins: [
+    // ...
+    plugin(({ addVariant }) => {
+      // Add variant for electronics store
+      addVariant("electronics", ".electronics &");
+      // Add variant for apparel store
+      addVariant("apparel", ".apparel &");
+    }),
+  ],
+  // ...
+} as Config;
+```
+
+Then use these variants (`electronics:`, `apparel:`, etc.) in your components:
+
+```vue
+// apps/storefront-unified-nuxt/components/ui/NavbarTop.vue
+<template>
+  <header
+    :class="[
+      'sticky top-0 z-40 flex h-14',
+      'bg-primary-800',
+      'electronics:!bg-primary-600',
+      'apparel:!bg-primary-400',
+      'md:-top-5 md:h-20 md:pt-2.5',
+    ]"
+  >
+    <slot />
+  </header>
+</template>
+```
+::
+
+::tip Using CSS Variables with Tailwind
+The CSS variables we defined can be used with Tailwind's color system. For example, `bg-primary-500` will use the store-specific primary color defined in the CSS variables. For more information, see the [Tailwind CSS documentation](https://v3.tailwindcss.com/docs/customizing-colors#using-css-variables).
+::
+
+::tip Understanding Generated CSS
```

Comment:
This one is also repeated (I already read it on line 206). However, this is the location where I would expect this tip so the first one should go and this one should stay 👍

Reply:
ah, sorry I've resolved the conflicts incorrectly